pkg/mock/generator.go

```diff
@@ -1,3 +1,6 @@
 package mock
 
 //go:generate minimock -g -i github.com/instill-ai/component/pkg/base.UsageHandler -o ./ -s "_mock.gen.go"
+//go:generate minimock -g -i ./../operator/pdf/v0.commandRunner -o ./ -s "_mock.gen.go"
```

Comment:
@jvallesm 
Before we have the component remotely, how do you set the minimock last time?
It seems that I have to add this back after we release the code.
Is the understanding correct?

Reply:
Except this part, others are ok. So, I request review first!